tests/plugins/chart/basic_chart.test.ts

```diff
@@ -1588,52 +1593,113 @@ describe("Chart design configuration", () => {
+      createChart(model, { ...defaultChart, type: chartType as "bar" | "line" | "pie" }, "42");
+      const runtime = model.getters.getChartRuntime("42") as BarChartRuntime;
+      const data = runtime.chartJsConfig.data!;
+      const tooltipItem = getTooltipItem(
+        data.datasets![0].data![0] as number,
+        chartType,
+        runtime.chartJsConfig
+      );
+      expect(
+        // @ts-ignore should be binded to the chart tooltip model
+        runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
+      ).toEqual("$6,000.00");
+    });
 
-    test.each(["bar", "line", "pie"])(
-      "Basic chart tooltip label, date format is ignored",
-      (chartType) => {
-        setCellContent(model, "A2", "6000");
-        setCellFormat(model, "A2", "m/d/yyyy");
-        createChart(model, { ...defaultChart, type: chartType as "bar" | "line" | "pie" }, "42");
-        const runtime = model.getters.getChartRuntime("42") as BarChartRuntime;
-        const data = runtime.chartJsConfig.data!;
-        const tooltipItem = getTooltipItem(data.datasets![0].data![0] as number, chartType);
-        expect(
-          // @ts-ignore should be binded to the chart tooltip model
-          runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
-        ).toEqual("6,000");
-      }
-    );
+    test.each(["bar", "line"])("Basic chart tooltip label, date format is ignored", (chartType) => {
+      setCellContent(model, "A2", "6000");
+      setCellFormat(model, "A2", "m/d/yyyy");
+      createChart(model, { ...defaultChart, type: chartType as "bar" | "line" | "pie" }, "42");
+      const runtime = model.getters.getChartRuntime("42") as BarChartRuntime;
+      const data = runtime.chartJsConfig.data!;
+      const tooltipItem = getTooltipItem(
+        data.datasets![0].data![0] as number,
+        chartType,
+        runtime.chartJsConfig
+      );
+      expect(
+        // @ts-ignore should be binded to the chart tooltip model
+        runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
+      ).toEqual("6,000");
+    });
+  });
+  describe("Pie Chart tooltip", () => {
+    test("pie chart tooltip label to include percentage", () => {
+      setCellContent(model, "A1", "P1");
+      setCellContent(model, "A2", "60000000");
+      createChart(
+        model,
+        {
+          dataSets: ["A1:A2"],
+          labelRange: "A1",
+          type: "pie",
+        },
+        "1"
+      );
+      const chart = model.getters.getChartRuntime("1") as PieChartRuntime;
+      const data = chart.chartJsConfig.data!;
+      const tooltipItem = getTooltipItem(
+        data.datasets![0].data![0] as number,
+        "pie",
+        chart.chartJsConfig
+      );
+      const expectedLabel = "60,000,000 (100.00%)";
+
+      expect(
+        // @ts-ignore
+        chart.chartJsConfig!.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
+      ).toBe(expectedLabel);
```

Comment:
It would improve the readability to replace the `getTooltipItem` helper by an helper that takes the chart runtime, a dataset index (if we use another dataSet than 0 in some tests), and a value index, and that directly returns the result of the `label()` callback. 

It would avoid having to compute a tooltipItem that we don't care about and to write a ts-ignore in each test.

You can change all the tests that use getTooltipItem